store-graphql/src/commonMain/kotlin/fr/androidmakers/store/graphql/GraphQLStore.kt

```diff
@@ -46,6 +46,10 @@ class GraphQLStore(val apolloClient: ApolloClient) : AndroidMakersStore {
         .toResultFlow()
   }
 
+  override fun getWifiInfo(): Flow<Result<WifiInfo?>> {
+    TODO("Not yet implemented")
```

Comment:
TODO

Reply:
This part of the codebase isn't used at the moment.